course/include/isometry.h

```diff
@@ -0,0 +1,78 @@
+#ifndef ISOMETRY_H
+#define ISOMETRY_H
+
+#include <cstdint>
+#include <iostream>
+#include <ostream>
+
+namespace ekumen {
+namespace math {
+
+class Vector3{
+    public:
+        // Constructors
+        Vector3() : x_(0), y_(0), z_(0) {};
+        Vector3(double x, double y = 0, double z = 0) : x_(x), y_(y), z_(z) {};
+        Vector3(const Vector3& v) : x_(v.x()), y_(v.y()), z_(v.z()) {} ;
+
+        // Destructor
+        ~Vector3() {}; // is this necessary?
+
+        // Operators
+        double& operator [] (const int);
+        double& operator [] (const int) const;
+        Vector3 operator + (const Vector3&) const;
+        Vector3 operator - (const Vector3&) const;
+        Vector3 operator * (const Vector3&) const;
+        Vector3 operator / (const Vector3&) const;
+        Vector3 operator = (const Vector3&);
+        Vector3& operator += (const Vector3&);
+        Vector3& operator -= (const Vector3&);
+        Vector3& operator *= (const Vector3&);
+        Vector3& operator /= (const Vector3&);
+        bool operator == (const Vector3&) const;
+        bool operator != (const Vector3&) const;
+        bool operator == (const std::initializer_list<double>& list) const;
+        bool operator != (const std::initializer_list<double>& list) const;
+
+        friend std::ostream& operator << (std::ostream& os, const Vector3& v) {
+            os << std::string("(x: ") << v.x() << ", y: " << v.y() << ", z: " << v.z() << ")";
+            return os;
+        }
+
+        // Getters
+        inline double x() const {return x_;};
```

Comment:
inline is not needed for in-class inline members. They are automatically so.

Reply:
Also remember that inline is only a suggestion to the compiler, but the compiler can ignore you.